File: class9/ex46_redoTheQuestions/class2/ex4_celsius_fahrenheit.c
/*Escreva um programa em C que converta uma temperatura em graus
Celsius para Fahrenheit. A fórmula para conversão é: Fahrenheit =
(Celsius * 9/5) + 32. O programa deve ler a temperatura em Celsius do
usuário e imprimir o resultado na tela.*/

#include <stdio.h>

int main() {

    float celsius;
    float fahrenheit;

    printf("Informe a temperatura em Celsius: ");
    scanf("%f", &celsius);

    fahrenheit = (celsius * 9 / 5) + 32;

    printf("Temperatura em Fahrenheit: %.2f\n", fahrenheit);

    return 0;
}

File: GO/class2/ex4_celsius_fahrenheit.go
/*Escreva um programa em C que converta uma temperatura em graus
Celsius para Fahrenheit. A fórmula para conversão é: Fahrenheit =
(Celsius * 9/5) + 32. O programa deve ler a temperatura em Celsius do
usuário e imprimir o resultado na tela.*/

package main

import "fmt"

func main() {

	var celsius, fahrenheit float64

	fmt.Print("Informe a temperatura em Celsius: ")
	fmt.Scan(&celsius)

	fahrenheit = (celsius * 9 / 5) + 32

	fmt.Printf("Temperatura em Fahrenheit: %.2f\n", fahrenheit)
}
